pdns/doh.hh

```diff
@@ -48,11 +48,22 @@ struct DOHFrontend
   std::string d_ciphers13;
   std::string d_serverTokens{"h2o/dnsdist"};
   LibsslTLSVersion d_minTLSVersion{LibsslTLSVersion::TLS10};
+#ifdef HAVE_DNS_OVER_HTTPS
+  std::unique_ptr<OpenSSLTLSTicketKeysRing> d_ticketKeys{nullptr};
+#endif
   std::vector<std::pair<std::string, std::string>> d_customResponseHeaders;
   ComboAddress d_local;
 
   uint32_t d_idleTimeout{30};             // HTTP idle timeout in seconds
   std::vector<std::string> d_urls;
+  std::string d_ticketKeyFile;
```

Comment:
Maybe also wrap these new members in a `#ifdef HAVE_DNS_OVER_HTTPS #endif` block ?

Reply:
I don't like having a lot of `#ifdef` block around, so as long as it doesn't prevent compiling when DoH support is disabled I'd rather leave them alone.